docs/operating-scylla/procedures/cluster-management/create-cluster.rst

```diff
@@ -11,15 +11,13 @@ Prerequisites
 Procedure
 ---------
 
-These steps need to be done for each of the nodes in the new cluster.
-
-1. Install ScyllaDB on a node. See :doc:`Getting Started</getting-started/index>` for further instructions.
+1. Install ScyllaDB on the nodes you want to add to the cluster. See :doc:`Getting Started</getting-started/index>` for further instructions.
 Follow the ScyllaDB install procedure up to ``scylla.yaml`` configuration phase.
 
 In case the node starts during the process, follow :doc:`these instructions </operating-scylla/procedures/cluster-management/clear-data>` 
 
-2. In the ``scylla.yaml`` file, edit the parameters listed below.
-The file can be found under ``/etc/scylla/``
+2. On each node, edit the ``scylla.yaml`` file to configure the parameters listed below.
+The file can be found under ``/etc/scylla/``.
 
 - **cluster_name** - Set the selected cluster_name
 - **seeds** - Specify the IP of the first node and **only the first node**. New nodes will use the IP of this seed node to connect to the cluster and learn the cluster topology and state.
```

Comment:
The concept of "first node" is no longer well defined given those changes -- we no longer assume that nodes are booted sequentially, so there is no "first" node.

We should introduce a step at the beginning of the procedure where we pick one of the nodes as the seed that will be used by all nodes and provided here.